### DISCUSSION TITLE
Detect language only without transcription

### DISCUSSION THREAD

**Discussion by mliradelc** [openai/whisper#962] [Q&A] [answered]
Hi, 

Is  it possible with the command line get only the detected language?. I know that you can do it with Python, but for the problem I'm working, adding a Python file would add more complexity.

Best regards

**Comment by glangford** [comment, marked as answer]
`whisper` doesn't currently have the option to only detect the language from the command line. You could try to implement something tricky to capture the output from the command line, look for "Detected language:", print that, then kill the `whisper` process before it finishes transcribing. But this would be even more complicated than creating a simple python script.
